plugins/wazuh-fleet/public/components/agents/details/dashboard/dashboard.tsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
-import { getPlugins } from '../../../../plugin-services';
 import { EuiSpacer } from '@elastic/eui';
+import { getPlugins } from '../../../../plugin-services';
 import './dashboard.scss';
 import { EventsCount } from './events-count';
 
 export interface AgentDashboardProps {
   agentId: string;
+  indexPattern: any;
+  filters: any[];
```

Comment:
The indexPattern prop is typed as any. Using any defeats the purpose of TypeScript. Consider defining a specific type for indexPattern to improve type safety and maintainability.

import { IndexPattern } from '../../../../data/public';

```suggestion
  indexPattern: IndexPattern;
```

Reply:
Solved https://github.com/wazuh/wazuh-dashboard-plugins/commit/cb6d1a6563111b7f559dae57bf9b10814f91bcfd